### ISSUE TITLE
InsertRow: unsupported relationship xl/calcChain.xml

### ISSUE THREAD

**Issue by l0rda** [baliance/gooxml#215]
## Description

There is another problem with InsertRow.

In our document we have cell D3, that contains any formula, that references D2, for example `=SUM(D2:D2)`.

After insert new row with number 2 i got error:

> unsupported relationship xl/calcChain.xml http://schemas.openxmlformats.org/officeDocument/2006/relationships/calcChain

xl/calcChain.xml content:
```<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<calcChain xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><c r="D3" i="1" l="1"/></calcChain>
```

I suppose that cell D2 is now D3(and contains number value), but calcChain.xml still contains formula for D3.

Working sample https://github.com/l0rda/testgooxmlinsert

**Comment by l0rda** [comment]
I tried to clear cell formula with `cell.Clear()` before inserting new row.
It does not help. I still see error:

![2018-10-16 16 49 28](https://user-images.githubusercontent.com/2686935/47022582-7c796500-d166-11e8-96db-4fe32586d2ab.png)

Also i tried to clear cells with formula without inserting new rows. And still get this error.
`cell.Clear()` does not correctly clear formula cells?

I tried to use `SetString("")` and `SetNumber(0)` on formula cell without effect. Excel still shows error on file open.
